book/src/examples/calculator.md

```diff
@@ -0,0 +1,139 @@
+# Simple calculator
+
+When you implement an interpreter for a DSL or a programming language, the process typically involves the following steps:
+
+1. **Lexing**: Splitting the input into tokens via a lexer.
```

Comment:
```suggestion
1. **Lexing**: Splitting the input stream (i.e., code) into tokens via a lexer.
```

Reply:
I partially applied the suggestion.

Suggested: `Splitting the input stream (i.e., code) into tokens`

Applied: `Splitting the input stream (i.e., source code string) into tokens`

Reply:
Looks great!